### ISSUE TITLE
[DOC] Docker images refer to an older version of dashboards

### ISSUE THREAD

**Issue by ashwin-pc** [opensearch-project/documentation-website#2234]
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [x] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
The docker files on the docsite point to an older version of opensearch and opensearch-dashboards, version 1.0.1. We should instead change that to 

```yml
image: opensearchproject/opensearch:latest

and 

image: opensearchproject/opensearch-dashboards:latest
```

https://opensearch.org/docs/1.0/opensearch/install/docker#sample-docker-compose-file

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

**Comment by Naarcha-AWS** [comment]
@ashwin-pc: The link you provided is for OpenSearch version 1.0. Do you run into the same problem using the latest version of the docs?

**Comment by ashwin-pc** [comment]
Oh then we have a whole different issue. I reached these docs when i was googling how to install OSD using docker and this was the top link. When i tried again just now it took me to 1.3.x and unless i look specifically at the version, its not obvious to me that im on an older version of the docs

**Comment by hdhalter** [comment]
I think this is a one-off. I found this one too, but it was not the first link. The first link was the topic on Docker that Jeff wrote and published a couple of weeks ago, so that's great news that its already been indexed and showing up as the first link. Most of the links on the first page are taking me to latest versions. Unless we want to remove earlier versions of documents from being indexed by Google, we may have to live with this. Especially if we want people to find older versions of our documentation, which we do because we know people are still using older versions.

**Comment by ashwin-pc** [comment]
@hdhalter yeah i'm okay with that but can we atleast add a banner on top of the old doc pages to indicate that they are looking at an older version of the docs? Even if i select 2.4 as the version i want to look at, its not obvious to me what the latest version is until i click the dropdown to see the available versions. A big banner on top of the site indicating that a newer version is available would solve this issue for now. 

e.g. 
<img width="1902" alt="Screen Shot 2022-12-16 at 5 02 36 PM" src="https://user-images.githubusercontent.com/20453492/208214937-18e98ab9-1932-41fd-b31e-bb6e1c06eb74.png">

I create the 3 issues linked here because I was doing a dry run of the process i would 
follow to setup a cluster myself for my personal use and these are the links i landed on and followed, so it is very likely that others are landing on these pages too and are completely unaware that they are looking at an earlier version of the docs.

**Comment by hdhalter** [comment]
Thanks, Ashwin. We are working on a solution for something similar (for versions we are no longer maintaining), so we will look into this, too. See https://github.com/opensearch-project/documentation-website/issues/2166.

**Comment by hdhalter** [comment]
Fanit is going to add a note in the banner for everything after 1.3 and a different one for everything before 1.3 (see #2166).

**Comment by kolchfa-aws** [comment]
Closed with #2641
